ui/ssb-msg.js

```diff
@@ -157,6 +155,20 @@ Vue.component('ssb-msg', {
     const profiles = SSB.db.getIndex('profiles').getProfiles()
     this.name = getName(profiles, this.msg.value.author)
 
+    // Render the body, which may need to wait until we're connected to a peer.
+    var self = this
+    const blobRegEx = /!\[.*\]\(&.*\)/g
+    if(self.msg.value.content.text.match(blobRegEx)) {
+      // It looks like it contains a blob.  There may be better ways to detect this, but this is a fast one.
+      // We'll display a sanitized version of it until it loads.
+      self.body = md.markdown(self.msg.value.content.text.replaceAll(blobRegEx, 'Loading...'))
```

Comment:
can we use a isConnected thing here so we don't first add the loading body and then the real body in case we are connected?